warp/src/main/java/com/schibsted/nmp/warp/theme/WarpDimensions.kt

```diff
@@ -38,6 +40,36 @@ object WarpDimensions {
     val components = WarpComponentDimensions
 
     val icon = WarpIconDimensions
+
+
+    /**
+     * Helper methods contributed by Shiva Bernhard
+     */
+    fun calculateDimensionResource(context: Context, resId: Int): Dp {
```

Comment:
I think it would make sense to include the official minimum touch size dimension in there>
`    fun minimumTouchSize(context: Context): Dp {
        return calculateDimensionResource(
            context = context,
            resId = R.dimen.minimum_touch_size
        )
    }
`